target/linux/ath79/image/nand.mk

```diff
@@ -30,6 +30,21 @@ define Build/zyxel-factory
 		fi
 endef
 
+define Device/8dev_rambutan
+  SOC := qca9557
+  DEVICE_VENDOR := 8devices
+  DEVICE_MODEL := Rambutan
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2
```

Comment:
kmod-usb-core can be dropped here, it will be pulled automatically by usb2.

Reply:
Fixed.